src/pages/[platform]/sdk/index.mdx

```diff
@@ -5,7 +5,8 @@ export const meta = {
   description:
     'The AWS Mobile SDK for Android enables you to build mobile apps by providing simplified APIs for AWS services such as Amazon Cognito, Amazon S3, AWS AppSync, and more.',
   disableTOC: 'true',
-  platforms: ['swift', 'android']
+  platforms: ['swift', 'android'],
+  hideChildrenOnBase: 'true'
```

Comment:
adding flag to sdk and cli pages that they should not render their children on the base menu